File: ProjectEuler10.py
from sympy import *
#Find the sum of all the primes below two million.

def sum_or_primes(x):
    list_of_primes = []
    for i in range(x):
        if isprime(i):
            list_of_primes.append(i)

    print(sum(list_of_primes))

sum_or_primes(2000000)

File: ProjectEuler10.rb
#find the sum of all the primes below 200000

require 'prime'

def sum_of_primes(x)
  list_of_primes = []
  (0..x).each do |num|
    if num.prime?
      list_of_primes.append(num)
    end
  end
  sum_list_of_primes = list_of_primes.sum
  puts sum_list_of_primes
end



sum_of_primes(2000000)
